### PR TITLE
Address #2493 Update 'genital system'

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Address #2493 Update 'genital system'

### CODE REVIEW

**Comment by shawntanzk** [comment]
@bvarner-ebi - if you dont need the diff, just ignore it. If you want it try re-running later (or do it manually yourself on your local machine if you know how to). Basically it's just resource issues I think.

**Review by ubyndr — APPROVED** [review]

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -10024,6 +10024,7 @@ xref: Wikipedia:Gonad
 xref: XAO:0003146
 xref: ZFA:0000413
 is_a: UBERON:0003133 ! reproductive organ
+is_a: UBERON:3011048 ! genital system
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:10027]: I think part_of would be better.

Comment (dosumis) [reply src/ontology/uberon-edit.obo:10027]: Also for most of all other children.  Same scope for refactoring to clean up too given hidden GCIs.

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**